### PR TITLE
Fix #514: Show only one TMO dashboard link in probe-dictionary

### LINKED ISSUE #514: Show only one TMO dashboard link in probe-dictionary
Thanks to the tabbed interface from #409, we can now treat the TMO dashboard as one single dashboard.

That means that in detail views in probe-dictionary like [this one](https://telemetry.mozilla.org/probe-dictionary/?detailView=histogram%2FA11Y_CONSUMERS), we can just put one single link called "Measurements dashboard". This should link to the `dist.html` dashboard.

### PR DESCRIPTION
**Pull request by Insiyaa**
A single link to measurements dashboard.
Link: https://insiyaa.github.io/telemetry-dashboard/probe-dictionary/

### CODE REVIEW

File: probe-dictionary/explore.js [hunk 1]
```diff
@@ -681,9 +681,7 @@ function getDatasetInfos(probeId, channel, state) {
     const distURL = getTelemetryDashboardURL('dist', probe.name, probe.type, channel, versions.first, versions.last);
     const evoURL = getTelemetryDashboardURL('evo', probe.name, probe.type, channel, versions.first, versions.last);
     datasetInfos.push("TMO dashboard: "
```
Comment (georgf) [inline probe-dictionary/explore.js]: We can remove the `"TMO dashboard: "` part now, it's not needed anymore.

File: probe-dictionary/explore.js [hunk 2]
```diff
@@ -681,9 +681,7 @@ function getDatasetInfos(probeId, channel, state) {
     const distURL = getTelemetryDashboardURL('dist', probe.name, probe.type, channel, versions.first, versions.last);
     const evoURL = getTelemetryDashboardURL('evo', probe.name, probe.type, channel, versions.first, versions.last);
```
Comment (georgf) [inline probe-dictionary/explore.js]: `evoURL` should be unused now, lets remove it.

**Review by georgf — COMMENTED** [review]
This is close, thanks!
I have just two change requests below.

**Comment by Insiyaa** [comment]
Done!

**Review by georgf — APPROVED** [review]
This looks good, thanks!

**Pull request merged**